app-modules/task/tests/EditTaskTest.php

```diff
@@ -53,7 +54,9 @@
 test('EditTask is gated with proper access control', function () {
```

Comment:
Because of the nature of the change here this test will need to be modified are changed to ensure that a User must have the `task.view-any` and `task.*.update` permission and either be the creator or currently assigned.

We need to test that all scenarios are covered and the proper response is returned.